libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -603,27 +536,16 @@ private Optional<List<PassportDetails>> generatePassportClaim(List<VerifiableCre
             var passport = credentialSubject.getPassport();
 
             if (isNullOrEmpty(passport)) {
-                StringMapMessage mapMessage =
-                        new StringMapMessage()
-                                .with(
-                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                        "Passport property is missing from VC or empty passport property.")
-                                .with(
-                                        LOG_CRI_ISSUER.getFieldName(),
-                                        passportVc.get().getCri().getId());
-                LOGGER.warn(mapMessage);
```

Comment:
These should not be warnings - lots of DCMAW VCs don't have a passport...

In the last week there were something like 20,000 of these type of warnings logged!